CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - `DataGrid`: the `componentWillReceiveProps` lifecycle is replaced, as it will be decrecated in react v17 ([@LennertBlommaert](https://github.com/LennertBlommaert) in [#331](https://github.com/teamleadercrm/ui/pull/331))
 - `Overlay`: the `componentWillUpdate` lifecycle is replaced, as it will be decrecated in react v17 ([@LennertBlommaert](https://github.com/LennertBlommaert) in [#331](https://github.com/teamleadercrm/ui/pull/331))
+- `Menu`: the `componentWillUpdate` and `componentWillRecieveProps` lifecycles are replaced, as they will be decrecated in react v17 ([@LennertBlommaert](https://github.com/LennertBlommaert) in [#330](https://github.com/teamleadercrm/ui/pull/330))
```

Comment:
A couple of remarks:

- there is `decrecated` 3 times in this file (typo)
- shouldn't we provide a bit more info? Replaced in favour of what?
- `React 17` not `react v17`

Reply:
Something like:
`Menu`: the `componentWillUpdate` and `componentWillRecieveProps` lifecycles are replaced by the `getDerivedStateFromProps` and `componentDidUpdate` lifecycles, as they will be deprecated in React 17 ([@LennertBlommaert](https://github.com/LennertBlommaert) in [#330](https://github.com/teamleadercrm/ui/pull/330))

For providing a bit more info?

Reply:
What @duivvv means is also add what it has changed into; `componentWillReceiveProps` ➡️ `aBetterLifecycleMethod`

Reply:
Yeah either one is fine, just a bit more info. @nickwaelkens solution might be a bit more concise.